husky_gazebo/launch/spawn_husky.launch

```diff
@@ -53,6 +55,17 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
     <!-- Stow the arm -->
     <node pkg="husky_control" type="stow_ur5" name="stow_ur5"/>
 
+    <!-- Start the pr2 gripper controller and action server -->
+    <group if="$(arg gripper_enabled)">
+        <rosparam command="load" file="$(find husky_control)/config/control_gripper.yaml" />
+        <node name="r_gripper_controller_spawner" pkg="controller_manager" type="spawner" args="r_gripper_controller"/>
+        <!-- https://github.com/ros-controls/ros_controllers/pull/181 -->
+        <node name="r_gripper_state_publisher" pkg="rostopic" type="rostopic"
```

Comment:
not sure, but maybe there is not transmission for gripper joints

--
◉ Kei Okada


On Fri, Jan 22, 2016 at 1:02 AM, Paul Bovbel <notifications@github.com>
wrote:

> In husky_gazebo/launch/spawn_husky.launch
> <https://github.com/husky/husky_simulator/pull/24#discussion_r50420034>:
>
> > @@ -53,6 +55,17 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
> >      <!-- Stow the arm -->
> >      <node pkg="husky_control" type="stow_ur5" name="stow_ur5"/>
> >
> > +    <!-- Start the pr2 gripper controller and action server -->
> > +    <group if="$(arg gripper_enabled)">
> > +        <rosparam command="load" file="$(find husky_control)/config/control_gripper.yaml" />
> > +        <node name="r_gripper_controller_spawner" pkg="controller_manager" type="spawner" args="r_gripper_controller"/>
> > +        <!-- https://github.com/ros-controls/ros_controllers/pull/181 -->
> > +        <node name="r_gripper_state_publisher" pkg="rostopic" type="rostopic"
>
> Understood, so then why is this necessary? Shouldn't the joint state
> controller publish the states for any controller joints?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/husky/husky_simulator/pull/24/files#r50420034>.
>